tools/js-sdk-release-tools/package.json

```diff
@@ -2,8 +2,15 @@
   "name": "@azure-tools/js-sdk-release-tools",
   "version": "2.7.11",
   "description": "",
+  "workspaces": [
+    "workspaces/*"
```

Comment:
nit: i feel like we mix the monorepo with workspaces and `src` with single project. Could we either organize them as pure monorepo or move our detector to src folder to keep single project?

Reply:
The goal is to setup as monorepo finally. but unfortunatly, there's two issues limit to convert to monorepo:
1. modular support PR is in review, if we change to monorepo, it's hard to the modular support pr later
2. the new breaking change detection tool has it's own setup like eslint, prettier and more. check in to src will make it hard to seperate it later.
We can setup it as mono repo after there's no pending PRs.